lib/lita/services/market_manager.rb

```diff
@@ -134,6 +139,23 @@ def build_new_order(user:, type:, created_at:, price:)
           price: price
         }
       end
+
+      def persist_transaction(buyer,seller,price)
+        sw = Lita::Services::SpreadsheetManager.new(
+          ENV.fetch('GOOGLE_SP_DB_KEY'),
+          'transactions'
+        )
+        sw.insert_new_row(
+          [
+            Time.now.strftime('%Y-%m-%d'),
```

Comment:
Do not use Time.now without zone. Use one of Time.zone.now, Time.current, Time.now.in_time_zone, Time.now.utc, Time.now.getlocal, Time.now.xmlschema, Time.now.iso8601, Time.now.jisx0301, Time.now.rfc3339, Time.now.httpdate, Time.now.to_i, Time.now.to_f instead.